addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -120,7 +120,6 @@ var SnippetEditor = Widget.extend({
     template: 'web_editor.snippet_overlay',
     xmlDependencies: ['/web_editor/static/src/xml/snippets.xml'],
     events: {
-        'click .oe_snippet_remove': '_onRemoveClick',
```

Comment:
I don't think your spec is to remove the delete button on all snippets

Reply:
I did not write this pull request.

Reply:
:roll_eyes: